experiments/optimizer/README.md

```diff
@@ -132,14 +133,15 @@ Check
 [DOT's GitHub Repo](https://github.com/adobe/aem-dispatcher-optimizer-tool/tree/main/plugin#configuration)
 for details on the different options for the Report Verbosity.
 
-For simple Dispatcher's there may actually be little difference.  For real world configurations, it can
+For relatively simple Dispatcher configurations there may actually be little difference.  
+For real world configurations, it can
 reduce the report rows by 100's of (repeated) violations, making the list easier to act upon.
 
 ## Test #5 - Overlay some rules
 
 Hopefully the value of the DOT is starting to become clear.  You may start to imagine new rules that you would like
-to implement.  Or you may notice a rule and decide, after very careful consideration, that you want to silence
-it (usually not recommended) or change its severity.  Let's see how to do that.
+to implement.  Or you may notice a rule and decide, after very careful consideration, that you want to 
+change the value being tested, silence it (usually not recommended), or change its severity.  Let's see how to do that.
```

Comment:
Which `value`?  The element?  filterValue?

Reply:
All of the above; in statfileslevel's case it's `value`, but `ruleValue` and `filterValue` are options for other Checks.

How about, "...change the `value`, `ruleValue` or `filterValue` being tested..."